CHANGELOG.md

```diff
@@ -55,7 +55,7 @@
     * FIXED: HTML injection via unescaped attachment filename (#554)
     * FIXED: Password disabling option (#527)
   * **1.2.2 (2020-01-11)**
-    * CHANGED: Upgrading libraries to: bootstrap 3.4.1, DOMpurify 2.0.7, jQuery 3.4.1, kjua 0.6.0, Showdown 1.9.1 & SJCL 1.0.8
+    * CHANGED: Upgrading libraries to: bootstrap 3.4.1 (CSS), DOMpurify 2.0.7, jQuery 3.4.1, kjua 0.6.0, Showdown 1.9.1 & SJCL 1.0.8
```

Comment:
```suggestion
    * CHANGED: Upgrading libraries to: bootstrap 3.4.1, DOMpurify 2.0.7, jQuery 3.4.1, kjua 0.6.0, Showdown 1.9.1 & SJCL 1.0.8
```

Now also JS, is not it?

Reply:
Yes, that was added to the 1.2.2 section to clarify that only the CSS got upgraded, while in the next release the JS side of it gets addressed. IMO the old release notes, in hindsight, were misleading (myself included, and I must have been the one that wrote it).